lib/gui/app.js

```diff
@@ -154,6 +155,16 @@ app.run(() => {
         allowSleepUpdateCheck: currentReleaseType === release.RELEASE_TYPE.PRODUCTION
       });
     });
+
+  // If the error is a user error, then we don't want
+  // to bother the user each time he opens the app.
+  // See: https://github.com/resin-io/etcher/issues/1525
+  }).catch(errors.isUserError, (error) => {
```

Comment:
Should we also check the `title` of the error, so that we only swallow 'Unable to check for updates' errors, in case there are other errors which we _do_ want the user to see?